css/styles.css

```diff
@@ -9,8 +9,6 @@
 	min-height: 100px;
```

Comment:
I think this and the next line are not necessary any more (min- and max-height). Tested only on Firefox and Chrome.

Reply:
The `min-height` is still needed, but `max-height` can be dropped. Adjusted it in this PR.